map/map.js

```diff
@@ -0,0 +1,32 @@
+import loadProfile from '../common/load-profile.js';
+import quests from '../data/quest-data.js';
+import { getUser } from '../data/api.js';
+import createQuestLink from '../map/create-quest-link.js';
+import createCompletedQuest from '../map/create-completed-quest.js';
+import hasCompletedAllQuests from '../map/has-completed-all-quests.js';
+import isDead from '../common/is-dead.js';
+import createCompleteQuest from '../map/create-completed-quest.js';
+
+
+loadProfile();
+
+const user = getUser();
+
+if (isDead(user) || hasCompletedAllQuests(quests, user)) {
+    window.location = '../results';
+}
+
+const nav = document.getElementById('quests');
+
+for (let i = 0; 0 < quests.length; i++) {
```

Comment:
moved a few things around in this logic. take a look and compare 
`for (let i = 0; i < quests.length; i++) {
    const quest = quests[i];
    let questDisplay = null;

    questDisplay = createQuestLink(quests);
    nav.appendChild(questDisplay);

    if (user.completed[quest.id]) {
        questDisplay = createCompleteQuest(quest);
    }

}`